src/App.js

```diff
@@ -51,6 +54,7 @@ class App extends Component {
         <HeaderBar isTopPage />
         <ul id='catalog-image-list' className='mdc-image-list standard-image-list mdc-top-app-bar--fixed-adjust'>
           {this.renderListItem('Button', buttonImg, `${PUBLIC_URL}/button`)}
+          {this.renderListItem('Checkbox', checkboxImg, `${PUBLIC_URL}/checkbox`)}
```

Comment:
FYI you can reference the image URL directly here and don't need PUBLIC_URL anymore, and don't need the image import above. (You'll probably see when you merge with master)